modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import AggsWrapper from './AggsWrapper.js';
+import { css } from 'emotion';
+import {
+  toggleSQON,
+  inCurrentSQON,
+  replaceSQON,
+  removeSQON,
+} from '../SQONView/utils';
+import './BooleanAgg.css';
+
+const booleanValues = {
+  true: 'true',
+  false: 'false',
+};
+
+export default ({
+  field,
+  buckets,
+  handleValueClick = () => {},
+  isTrueSelected = () => false,
+  isFalseSelected = () => false,
+  ...rest
+}) => {
+  const trueBucket = buckets.find(({ key }) => key === '1');
+  const falseBucket = buckets.find(({ key }) => key === '0');
+  const dotField = field.replace(/__/g, '.');
+
+  const isTrueActive = isTrueSelected({
+    evaluateInSqon: ({ sqon }) =>
+      inCurrentSQON({
+        dotField,
+        value: booleanValues.true,
+        currentSQON: sqon,
+      }),
+  });
+  const isFalseActive = isFalseSelected({
+    evaluateInSqon: ({ sqon }) =>
+      inCurrentSQON({
+        dotField,
+        value: booleanValues.false,
+        currentSQON: sqon,
+      }),
+  });
+  const isNeitherActive = !isTrueActive && !isFalseActive;
+
+  const handleTrueClick = () =>
```

Comment:
do we really need both of these functions, and not just one function that takes a boolean argument?

Reply:
not sure why I kept two... yes, refactored!